exercises/1-the-manual-menace/README.md

```diff
@@ -83,6 +75,10 @@ https://codeready-workspaces.apps.<DOMAIN_FOR_YOUR_CLASS>/dashboard/#/load-facto
 
 ![che-workspace-create](../images/exercise1/che-workspace-create.png)
 
+<p class="tip">
+<b>NOTE</b> - If after a few minutes your workspace keeps in ´Initializing´ state, refresh your browser.
```

Comment:
There's a link to the bottom left of the screen - something like "Back to Dashboard". I typically like using that one instead of refreshing the browser. Maybe we can call this link out here as well?

Reply:
I don't see this as a blocker really, anything should work, adding a note as @jtudelag did, ading the info @oybed mentioned or even don't add anything and just let the audience know about this random issue the same way we remember everyone to use Google Chrome instead any other browser.

Reply:
@makentenza my suggestion was for some additional info/improvements that goes well with what @jtudelag already have added. Should be a win-win really.

Reply:
@oybed I didnt realize that link was there, but sure, we can mention that in the PR instead of refreshing the browser, I can update the PR.